src/Uno.Toolkit.Skia.WinUI/Controls/Shadows/ShadowContainer.Paint.cs

```diff
@@ -72,7 +72,7 @@ private void OnSurfacePainted(object? sender, SKPaintSurfaceEventArgs e)
 		using var _ = canvas.SnapshotState();
 
 		var key =
-			FormattableString.Invariant($"[{contentAsFE.ActualWidth}x{contentAsFE.ActualHeight},{_shadowHost.ActualWidth}x{_shadowHost.ActualHeight},{background}]: ") +
+			FormattableString.Invariant($"[{contentAsFE.ActualWidth}x{contentAsFE.ActualHeight},{_shadowHost.ActualWidth}x{_shadowHost.ActualHeight},{background},{shape}]: ") +
```

Comment:
can you add ToString method to various IShadowShapeContext's concrete classes?
to make them more concise than: 
> CornerRadiusRectShadowShapeContext { Width = 5, Height = 10, CornerRadius = 2,2,2,2 }

Reply:
Done. Thanks.